packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
-import { Async } from 'office-ui-fabric-react/lib/Utilities';
 import { createListItems, IExampleItem } from '@uifabric/example-data';
 import { IColumn, buildColumns, SelectionMode, Toggle } from 'office-ui-fabric-react/lib/index';
 import { ShimmeredDetailsList } from 'office-ui-fabric-react/lib/ShimmeredDetailsList';
+import { useSetInterval } from '@uifabric/react-hooks';
+
+export interface IShimmerApplicationExampleState {
+  lastIntervalId: number;
+  visibleCount: number;
+}
+
+const ITEMS_COUNT = 200;
+const INTERVAL_DELAY = 1200;
```

Comment:
Is the change from 2500 to 1200 here intended?

Reply:
yes. It is minor, but I think it makes it more obvious what's going on.